test/unit/orchestration/tftp_test.rb

```diff
@@ -19,6 +19,15 @@ def test_host_should_not_have_tftp
     end
   end
 
+  def test_host_without_pxe_loader_should_not_have_tftp
+    if unattended?
```

Comment:
you can instead just add `skip_without_unattended` and remove the if condition, looks cleaner imho